demos/gfql/ten_mins_to_gfql.ipynb

```diff
@@ -0,0 +1,2376 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 10 mins to GFQL: ICIJ FinCEN Files Visualization\n",
+    "\n",
+    "This notebook demonstrates GFQL (Graph Query Language) using the ICIJ FinCEN Files dataset.\n",
```

Comment:
* imagine someone who doesn't know with ICIJ FinCEN is -- what is it? "ICIJ FInCEN Files dataset about ...". What's the use case's narrative story?

* Add another sentence: This tutorial shows doing x, y, z  <-- why they should keep reading, what's the technical benefit shown (how to do abc)